src/autoyast-rnc/security.rnc

```diff
@@ -9,7 +9,9 @@ cwd_in_user_path = element cwd_in_user_path { STRING }
 disable_restart_on_update = element disable_restart_on_update { STRING }
 disable_stop_on_removal = element disable_stop_on_removal { STRING }
 extra_services = element extra_services { STRING }
-selinux_mode = element selinux_mode { STRING }
+selinux_mode = element selinux_mode { "permissive" | "enforcing" | "disabled" }
```

Comment:
so is it still deprecated at https://github.com/yast/yast-security/blob/master/src/lib/y2security/autoinst_profile/security_section.rb#L44 ?

Reply:
ah never mind. I overlook that here is also modification of that security section.

Reply:
yep, basically we introduced the **lsm** section in #115 but after the review it was decided to no expose so much functionality as it is done in the installation control file, so, as we only offer the option for selecting the desired **LSM** we also removed the new section